README.md

```diff
@@ -1,6 +1,17 @@
 # CSI NFS driver
 
-## Kubernetes
+## Overview
+
+This is a repository for [NFS](https://en.wikipedia.org/wiki/Network_File_System) [CSI](https://kubernetes-csi.github.io/docs/) Driver.
+Currently it implements bare minimum of the CSI and the state is alpha.
+
+#### CSI Feature matrix
+
+| **nfs.csi.k8s.io** | CSI versions compatibility| Dynamic Provisioning | Resize | Snapshots | Raw Block | AccessModes              | Status |
+|--------------------|---------------------------|----------------------|--------|-----------|-----------|--------------------------|--------|
+|master              | v1.0.0, v1.1.0            |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
+|v1.0.0              | v1.0.0, v1.1.0            |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
```

Comment:
We may have to go with a v1.1 or maybe even a 2.0? The old driver in the deprecated repo already has a 1.0.2: https://github.com/kubernetes-csi/drivers/releases